README.md

```diff
@@ -270,7 +292,7 @@ Title of section.
 
 #### `files` ####
 
-Type: [Standardized File Format](#standardized-file-format)
+Type: Array of [Standardized File Format](#standardized-file-format)
```

Comment:
Is there a reason for adding 'array'? In Standardized file format we state that it can be a string, array or object. Can't the user just give a string to a file here?